### PR TITLE
chore(tiflash): update build steps for tiflash starts v8.5.0

### PR DESCRIPTION
**Pull request by wuhuizuo**
Ref https://github.com/pingcap/tiflash/pull/9587, it changes the build script paths.

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: packages/packages.yaml.tmpl [hunk 1]
```diff
@@ -1627,11 +1627,58 @@ components:
       - if: {{ semver.CheckConstraint ">= 6.1.0-0, < 8.2.0-0" .Release.version }}
         image: hub.pingcap.net/ee/ci/release-build-base-tiflash:v20231106
     routers:
-      - description: For range [v8.4.0, )
+      - description: For range [v8.5.0, ) it changed the build script path.
         if: {{ semver.CheckConstraint ">= 8.4.0-0" .Release.version }}
```
Comment (wuhuizuo) [inline packages/packages.yaml.tmpl:1631]:
```suggestion
        if: {{ semver.CheckConstraint ">= 8.5.0-0" .Release.version }}
```

File: packages/packages.yaml.tmpl [hunk 2]
```diff
@@ -1627,8 +1627,55 @@ components:
...
+      - description: For range [v8.5.0, ) it changed the build script path.
+        if: {{ semver.CheckConstraint ">= 8.5.0-0" .Release.version }}
+        os: [linux, darwin]
+        arch: [amd64, arm64]
+        profile: [release, enterprise]
+        steps:
+          release:
+            - os: linux
+              script: |
+                ./release-rocky8-llvm/scripts/build-tiflash-release.sh
```
Comment (wuhuizuo) [inline packages/packages.yaml.tmpl:1639]: same as previous route,just changed the folder name.

**Review by Lloyd-Pottiger — APPROVED** [review]

**Comment by wuhuizuo** [comment]
May be we should make the script path more common. I hold it and convert it to draft state.

**Pull request merged**